Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Generic/Validation/BoilerPlantEnabler.mo

```diff
@@ -0,0 +1,134 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Generic.Validation;
+model BoilerPlantEnabler "Validation model for BoilerPlantEnabler sequence"
+  CDL.Continuous.Sources.Sine sin(
+    amplitude=2,
+    freqHz=1/(6*60),
+    offset=2,
+    startTime=1)
+    annotation (Placement(transformation(extent={{-92,66},{-72,86}})));
+  CDL.Conversions.RealToInteger reaToInt
+    annotation (Placement(transformation(extent={{-62,66},{-42,86}})));
+  CDL.Continuous.Sources.Sine sin1(
+    amplitude=2/1.8,
+    freqHz=1/700,
+    phase=3.1415926535898,
+    offset=273 + (80 - 32)/1.8,
+    startTime=1)
+    annotation (Placement(transformation(extent={{-90,30},{-70,50}})));
+  CDL.Continuous.Sources.Sine sin2(
+    amplitude=2,
+    freqHz=1/(6*60),
+    offset=2,
+    startTime=1)
+    annotation (Placement(transformation(extent={{-92,-38},{-72,-18}})));
+  CDL.Conversions.RealToInteger reaToInt1
+    annotation (Placement(transformation(extent={{-62,-38},{-42,-18}})));
+  CDL.Conversions.RealToInteger reaToInt2
+    annotation (Placement(transformation(extent={{40,66},{60,86}})));
+  CDL.Continuous.Sources.Sine sin5(
+    amplitude=2/1.8,
+    freqHz=1/700,
+    phase=3.1415926535898,
+    offset=273 + (80 - 32)/1.8,
+    startTime=1)
+    annotation (Placement(transformation(extent={{12,30},{32,50}})));
+  CDL.Conversions.RealToInteger reaToInt3
+    annotation (Placement(transformation(extent={{38,-38},{58,-18}})));
+  CDL.Continuous.Sources.Constant con(k=3)
+    annotation (Placement(transformation(extent={{10,66},{30,86}})));
+  CDL.Continuous.Sources.Constant con1(k=273 + (75 - 32)/1.8)
+    annotation (Placement(transformation(extent={{-92,-70},{-72,-50}})));
+  CDL.Continuous.Sources.Constant con2(k=3)
+    annotation (Placement(transformation(extent={{8,-38},{28,-18}})));
+  CDL.Continuous.Sources.Constant con3(k=273 + (75 - 32)/1.8)
+    annotation (Placement(transformation(extent={{8,-70},{28,-50}})));
+  Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Generic.BoilerPlantEnabler
```

Comment:
It is good practice to have these tested instances at the top of the model. Also please use final for parameters, as these tests validate the performance of the block and should not be changed by the user. Also remember to protect any of the blocks that are not being used by the `mos` script.